packages/components/src/components/CallToAction.js

```diff
@@ -60,34 +99,79 @@ const Button = styled(_Button)`
   ${buttonAlignStyles}
 `;
 
+const InnerContainer = styled.div`
+  ${t((_, { type }) =>
+    !(type === Type.HORIZONTAL)
+      ? null
+      : _.tabletMinQuery(css`
+          display: flex;
+          flex-wrap: wrap;
+          flex-direction: row;
+          align-items: center;
+          justify-content: space-between;
+          > * {
+            padding: 0;
+            margin: 0;
+          }
+          ${Heading} {
+            flex: 1 0;
+          }
+          ${Text} {
+            flex: 1 1 calc(50% - 200px);
+            padding: 0 20px;
+          }
+          a {
+            flex: 0 0 200px;
+            width: 20%;
+            ${Button} {
+              margin: 0;
+              width: 100%;
+            }
+          }
+        `),
+  )};
+`;
+
 export default function CallToAction({
   align = Align.LEFT,
   backgroundImage,
   buttonText,
   buttonUrl,
+  type = Type.VERTICAL,
   reveal,
+  spacing = Align.SMALL,
```

Comment:
this needs to be referencing spacing I suppose?

Reply:
@maartenbusstra yep, oops :)